src/platforms/kotlin-multiplatform/native-access-sdk.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Native Access to SDK
+sidebar_order: 8
+description: "How to directly access the SDK without the shared module."
+---
+
+If you need to access the SDK directly in your native platform without the shared module, you first need to change the dependency configuration from `implementation(...)` to `api(...)`:
```

Comment:
Maybe add the SDK as `api` directly instead? so this section isn't even needed.

Reply:
This might be confusing since using api only allows direct access to the sdk on java/android but not on iOS. That needs an extra setup in which the user needs to export the framework. 

The way the doc is currently set up (with api and exporting framework in the install docs) users will think this is mandatory to make the SDK work but that's just optional

Reply:
Alright, let's keep it for now, we can improve it later if needed.